lib/tsd-file-api-client/src/main/java/no/uio/ifi/tc/model/pojo/ResumableUpload.java

```diff
@@ -32,6 +32,6 @@ public class ResumableUpload extends TSDFileAPIResponse {
   @SerializedName("max_chunk")
   private BigInteger maxChunk;
 
-  @SerializedName("md5sum")
+  @SerializedName("md5Sum")
```

Comment:
This now works with the TSD stub code, but are we sure it's compatible with the real API? Could approving it here now make it break with the "real world"?

Reply:
Temporary fix with [eb97539](https://github.com/ELIXIR-NO/FEGA-Norway/pull/189/commits/eb97539c643fa3d77913c28ac5145d7ca1172418)